### PR TITLE
DGJ-821 Initial push for camunda DEV deployment

### PR DESCRIPTION
**Pull request by bhumin-fw**
## Summary

Change Camunda workflow. Generate build on github and push it to openshift.

## Changes
- Update forms-flow-bpm-cd file with required changes
- Add new environment specific param in git.
- Add new bpm_bc and bpm_dc files to support build and deployment config.

## Notes
Added separate files other than existing camunda deployment flow.

### CODE REVIEW

**Review by matbusby-fw — APPROVED** [review]
Works for me.  I'm curious why we're adding the step to print the environment variables.

**Comment by bhumin-fw** [comment]
> Works for me. I'm curious why we're adding the step to print the environment variables.

@matbusby-fw My bad forgot to remove dependancy in steps. We print env parallel in steps to keep an eye from which branch and on which environment it is deploying. No strong reason here.

File: .github/workflows/forms-flow-bpm-cd.yml [hunk 1]
```diff
@@ -1,92 +1,199 @@
...
+          test -n "${BRANCH}"
+          echo "Current ENV is ${ENVIRONMENT}"
+          echo "Current namespace is ${NAMESPACE}"
+          oc -n ${NAMESPACE} project
+          oc -n ${NAMESPACE} process -f bpm_dc.yaml \
+                -p CAMUNDA_ANALYTICS_JDBC_URL=${CAMUNDA_ANALYTICS_JDBC_URL} \
+                -p CAMUNDA_JDBC_USER=${CAMUNDA_JDBC_USER} \
+                -p FORMIO_FILE_URL="${CAMUNDA_FORMIO_FILE_URL}" \
+                -p TOOLS_WORKSPACE=${BUILD_NAMESPACE} \
+                -p SMTP_SERVER="apps.smtp.gov.bc.ca" \
```
Comment (iman-jamali-fw) [inline .github/workflows/forms-flow-bpm-cd.yml:175]: I suggest moving the hardcoded values to envs at top of the file to ensure one can edit the config in one place.

Comment (bhumin-fw) [reply .github/workflows/forms-flow-bpm-cd.yml]: Agree with your point Iman, Move it to top of file along with env section.

**Review by iman-jamali-fw — APPROVED** [review]
Looks great! Thank you

**Pull request merged**